content/en/blog/_posts/2021-07-16-read-write-once-pod-access-mode-alpha.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: blog
+title: "Introducing Single Pod Access Mode for PersistentVolumes"
+date: 2021-07-16
+slug: read-write-once-pod-access-mode-alpha
+---
+
+**Author:** Chris Henzie (Google)
+
+Kubernetes 1.22 introduces a new ReadWriteOncePod access mode for PersistentVolumes and PersistentVolumeClaims that allows users to restrict volume access to a single pod in the cluster.
```

Comment:
Optionally: link PersistentVolumes and PersistentVolumeClaims to https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes and https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims (respectively)